README.md

```diff
@@ -1,6 +1,6 @@
 # WordPress for iOS #
 
-[![CircleCI](https://circleci.com/gh/wordpress-mobile/WordPress-iOS.svg?style=svg)](https://circleci.com/gh/wordpress-mobile/WordPress-iOS)
+[![Build status](https://badge.buildkite.com/2f3fbb17bfbb5bba508efd80f1ea8d640db5ca2465a516a457.svg)](https://buildkite.com/automattic/wordpress-ios)
```

Comment:
We could also remove this wholesale, given GitHub reports the build status for each commit these days, something that didn't happen at the time the badges became trendy.